docs-content/general/changelog/changelog-19.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Changelog 19 (05/22)
+slug: changelog-19
+---
+
+## Demo Project
+
+Check out [https://app.highlight.io/demo](https://app.highlight.io/demo) to check out our new demo project.
```

Comment:
Maybe intentional, but do we want to say "check out" twice in this sentence or should we come up with another way of saying this?

Also, do you think we should mention anything about what the project is (our marketing site)? Maybe we could even say something about people being able to find their own sessions on our marketing site if they look closely.

Reply:
oooOOooOOO.

Thanks for catching this terrible copy. I'm fixing it now.

I'll try to come up with something fun around finding one's own sessions.